lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -14,6 +14,7 @@
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
 package OpenQA::Worker::Engines::isotovideo;
+use Mojo::Base 'Mojo::EventEmitter', -signatures;
 
 use strict;
 use warnings;
```

Comment:
try

```suggestion
use Mojo::Base 'Mojo::EventEmitter', -base, -signatures;
```

see https://docs.mojolicious.org/Mojo/Base#DESCRIPTION

Reply:
Thanks. I removed this line. I agree with your words `not mix features, bugfixes and signature updates`